typescript/infra/config/grafanaAlerts.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  HIGH_URGENCY_RELAYER_FOOTER,
+  HIGH_URGENCY_RELAYER_HEADER,
+  LOW_URGENCY_KEY_FUNDER_FOOTER,
+  LOW_URGENCY_KEY_FUNDER_HEADER,
+} from '../scripts/funding/utils/alert-query-templates.js';
+import {
+  BalanceThresholdType,
+  balanceThresholdConfigMapping,
+} from '../scripts/funding/utils/constants.js';
+
+export enum AlertType {
+  LowUrgencyKeyFunderBalance = 'lowUrgencyKeyFunderBalance',
+  LowUrgencyEngKeyFunderBalance = 'lowUrgencyEngKeyFunderBalance',
+  HighUrgencyRelayerBalance = 'highUrgencyRelayerBalance',
+}
+
+interface AlertConfig {
+  walletName: WalletName;
+  grafanaAlertId: string;
+  configFileName: string;
+  choiceLabel: string;
+  queryTemplate: {
+    header: string;
+    footer: string;
+  };
+}
+
+export enum WalletName {
+  KeyFunder = 'keyFunder',
+  Relayer = 'relayer',
+  // ATAPayer = 'ataPayer',
+}
+
+export const walletNameQueryFormat: Record<WalletName, string> = {
+  [WalletName.KeyFunder]: 'key-funder',
+  [WalletName.Relayer]: 'relayer',
+  // [WalletName.ATAPayer]: '.*ata-payer
+};
+
+export const alertConfigMapping: Record<AlertType, AlertConfig> = {
+  [AlertType.LowUrgencyKeyFunderBalance]: {
+    walletName: WalletName.KeyFunder,
+    grafanaAlertId: 'ae9z3blz6fj0gb',
+    configFileName:
+      balanceThresholdConfigMapping[
+        BalanceThresholdType.LowUrgencyKeyFunderBalance
+      ].configFileName,
+    choiceLabel:
+      balanceThresholdConfigMapping[
+        BalanceThresholdType.LowUrgencyKeyFunderBalance
+      ].choiceLabel,
```

Comment:
nit: any harm in doing this? think we end up passing along `dailyRelayerBurnMultiplier` as well but not a big deal right, if it makes this a little cleaner
```suggestion
    ...balanceThresholdConfigMapping[
      BalanceThresholdType.LowUrgencyKeyFunderBalance
    ],
```

Reply:
Agree that it looks cleaner but I think its best to avoid mixing configs where possible specially given that we may add more fields to `balanceThresholdConfigMapping` in future as this evolves

Reply:
sgtm